index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link href="https://fonts.googleapis.com/css2?family=roboto&display=swap" rel="stylesheet">
+<title>The Code Magazine</title>
+</head>
+<body>
+    <h1><img src="img/icons8-blue-book-96.png" style = "height: 30px; width:30px;" alt = "Blue book icon" > The Code Magazine</h1>
```

Comment:
THe book image was supposed to be an icon. Smth like this: "&#128216". 
But it's fine to be an image